scripts/usbdaemon_test/usbdaemon_test.rs

```diff
@@ -0,0 +1,146 @@
+#![feature(exit_status, path_ext)]
```

Comment:
Thoughts on almost fully duplicating this program vs adding an option to select between domain socket client/server, and pass in a text file with the command set?